tests/js/spec/views/organizationGroupDetails/groupDetails.spec.jsx

```diff
@@ -105,25 +103,26 @@ describe('groupDetails', () => {
       body: {firstRelease: group.firstRelease, lastRelease: group.lastRelease},
     });
   });
+
   afterEach(() => {
     ProjectsStore.reset();
     GroupStore.reset();
     GlobalSelectionStore.reset();
     MockApiClient.clearMockResponses();
   });
 
-  it('renders', () => {
+  it('renders', async function () {
     ProjectsStore.reset();
     const {findByText, queryByText} = createWrapper();
 
     expect(queryByText(group.title)).toBeNull();
 
     ProjectsStore.loadInitialData(organization.projects);
 
-    expect(findByText(group.title)).toBeTruthy();
+    expect(await findByText(group.title, {exact: false})).toBeTruthy();
```

Comment:
This seems like an easy mistake to make, is there a way to prevent this from happening?

e.g. can we write these assertions in a different way?

Reply:
Another alternative would be to write 
`expect(await findByText(title: ${group.title})).toBeTruthy();`

Reply:
I will check if there are more options 😉